values-schema.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-'$schema': 'http://json-schema.org/draft-07/schema'
+$schema: http://json-schema.org/draft-07/schema#
```

Comment:
Does this work?

Reply:
Well, my yaml language server is very inconsistent. However, it seems to be the same as quoted. I've converted both to json, with quotes:

```
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "additionalProperties": false,
  "definitions": {
```

without:

```
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "additionalProperties": false,
  "definitions": {
```

So I don't think we need quotes, unless I'm mistaken.